TCS34725.h

```diff
@@ -133,7 +133,7 @@ class TCS34725_
         bool b = read8(Reg::STATUS) & (uint8_t)Mask::STATUS_AINT;
         if (b)
         {
-            update();
+            readCRGB(&raw_data);
```

Comment:
No need to change signature

Reply:
I think I can finally update my pull request after clarifying this issue with you. The reason for me to change this was to be able to use a fully IRQ driven program flow. `update` reads a register to see if data is available, which I know it is when I use the external IRQ of the chip. Thus, I made `readCRGB` public (to avoid I2C access of `read8(Reg::STATUS)`).
To give you an idea in my program I use these calls
`  if (dat_avail) {            
    tcs.readCRGB(&data);
    tcs.clearInterrupt();
    dat_avail = false; ...`
and an IRQ handler that sets `dat_avail = true;`
Do you still want to keep it as it is?

Reply:
Now I also remember why I added the pointer as a parameter. Otherwise the calling program always requires a copy operation of the data, i.e. `TCS34725::RawData raw = tcs.raw();` which is unnecessary. Just a thought ...

Reply:
Sorry I've missed the change about making `update` (`readCRGB`) public. I think it's better to make apis better for  interrupt driven way, but it should be separated from this pull request. I think the main purpose of this pull request is reading data in one i2c request.

About passing the pointer, I think it's not necessary. But I agree with calculating rgb data only when you call `color()`.